src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/NuGetProjectTelemetryService.cs

```diff
@@ -64,21 +64,21 @@ public void EmitNuGetProject(NuGetProject nuGetProject)
 
                 // Get project type.
                 var projectType = NuGetProjectType.Unknown;
-                if (nuGetProject is MSBuildNuGetProject)
+                if (nuGetProject.ProjectStyle == ProjectModel.ProjectStyle.PackagesConfig)
                 {
```

Comment:
why not keep consistency either use type cast or project style? And if you use projectstyle then make sure it's being set correctly for all the projects.